app/views/editions/_actions.html.erb

```diff
@@ -2,41 +2,40 @@
   <div class="panel-body">
     <div class="btn-toolbar" role="toolbar">
 
-      <% if edition.persisted? %>
+      <% edition_policy = EditionPolicy.new(current_user, edition) %>
```

Comment:
At the moment it feels like a view layer concern to me. It's just involved with displaying buttons or not. This kind of thing can go into helpers but I don't think it would be a big win at the mo.

The controller would definitely be the right place for this thing if we were implementing these conditions server side.. Which we should be doing! :)